vulntotal/tests/test_snyk.py

```diff
@@ -52,6 +52,37 @@ def test_generate_package_advisory_url(self):
         ]
         util_tests.check_results_against_expected(results, expected)
 
+    def test_generate_purl(self):
```

Comment:
Add a few tests for `golang` too.

Reply:
Done. Have added additional tests for golang and scoped npm packages.